whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/dao/NrtmVersionInfo.java

```diff
@@ -0,0 +1,12 @@
+package net.ripe.db.nrtm4.client.dao;
+
+public record NrtmVersionInfo(Long id, String source, Long version, String sessionID, String type
```

Comment:
NrtmClientVersionInfo

Reply:
is not too redundant called all the objects client when they are already behing client module and package?

Reply:
Done